moto/rds2/models.py

```diff
@@ -27,19 +25,23 @@ def __init__(self, **kwargs):
         self.region = kwargs.get('region')
         self.engine = kwargs.get("engine")
         self.engine_version = kwargs.get("engine_version", None)
-        self.default_engine_versions = {"MySQL": "5.6.21",
-                                        "mysql": "5.6.21",
-                                        "oracle-se1": "11.2.0.4.v3",
-                                        "oracle-se": "11.2.0.4.v3",
-                                        "oracle-ee": "11.2.0.4.v3",
-                                        "sqlserver-ee": "11.00.2100.60.v1",
-                                        "sqlserver-se": "11.00.2100.60.v1",
-                                        "sqlserver-ex": "11.00.2100.60.v1",
-                                        "sqlserver-web": "11.00.2100.60.v1",
-                                        "postgres": "9.3.3"
-                                        }
-        if not self.engine_version and self.engine in self.default_engine_versions:
-            self.engine_version = self.default_engine_versions[self.engine]
+
```

Comment:
if the port was not specified in the request, moto was returning None instead of the default port for the selected engine (which triggered and exception in the client libraries)